src/java.base/share/conf/security/java.security

```diff
@@ -775,7 +775,10 @@ http.auth.digest.disabledAlgorithms = MD5, SHA-1
 # This is in addition to the jdk.certpath.disabledAlgorithms property above.
 #
 # See the specification of "jdk.certpath.disabledAlgorithms" for the
-# syntax of the disabled algorithm string.
+# syntax of the disabled algorithm string. Additionally, TLS cipher suites
+# can be disabled here using "*" wildcard syntax. For example "TLS_RSA_*"
+# disables all cipher suites that start with "TLS_RSA_". Only the algorithms
```

Comment:
s/the algorithms/cipher suites/
s/have/have a/

Can you only have one wildcard? If so, we should also state that.